packages/state-manager/src/sagas/communities/updateCommunityMetadata/updateCommunityMetadata.saga.ts

```diff
@@ -35,13 +35,9 @@ export function* sendCommunityMetadataSaga(
     rootCa: community.rootCa,
   }
 
-  const meta = yield* apply(
+  yield* apply(
     socket,
     socket.emitWithAck,
     applyEmitParams(SocketActionTypes.SET_COMMUNITY_METADATA, communityMetadataPayload)
   )
-
-  if (meta) {
-    yield* put(communitiesActions.saveCommunityMetadata(meta))
-  }
```

Comment:
This is also replaced by a COMMUNITY_UPDATED event. Going forward, we can remove this saga entirely.